projects/fsastorefrontlib/src/core/sales-illustration/facade/chart.service.ts

```diff
@@ -0,0 +1,76 @@
+import { Injectable } from '@angular/core';
+import { TranslationService } from '@spartacus/core';
+import { EChartsOption } from 'echarts';
+import { combineLatest, Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+
+@Injectable()
+export class ChartService {
```

Comment:
This class is so specific it shouldn't be called `ChartService`. It cannot be reused in any other context except for savings...
Also since it's in `sales-illustration` directory, maybe it should be `sales-illustration-chart` or something similar.